### PR TITLE
Add status page references

### LINKED ISSUE #4697: Add Status Page references in the TTS console
#### Summary
We have status pages for the operational status of The Things Stack deployments (TTS Cloud & TTS Community Edition) and Packet Broker. Mention these references in the TTS console so that users will be aware and easily redirected to the respective status pages to know the status in case of any incidents.
- TTS Cloud: https://status.thethings.industries/
- TTS Community Edition: https://status.thethings.network/
- Packet Broker: https://status.packetbroker.net/

#### Why do we need this?
Users will be aware of the current operational status of the product.

#### What is already there? What do you see now?
References to the Documentation and Release notes at the footer section of the console.

#### What is missing? What do you want to see?
Status page references in the TTS console.
**Optional**: It would be good if we can add a configuration option in TTS to set the reference of the status page, such that self-hosted users can set the reference of their status page. 
cc: @michalborkowski96 

#### Environment
TTS V3.15.0
#### How do you propose to implement this?
...
#### How do you propose to test this?
...
#### Can you do this yourself and submit a Pull Request?
No @kschiffer

### PR DESCRIPTION
**Pull request by ryaplots**
#### Summary

References #4697 

#### Changes

1. Add necessary config and changes in the front-end to have references to:

- TTS Cloud status page
- TTS Community Edition status page

in the footer.

2. Add status page link to Packet Broker.
Screenshot 2022-03-23 at 10 19 29

#### Testing

Manually

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by htdvisser — APPROVED** [review]
LGTM. Please don't forget to create issues in https://github.com/TheThingsIndustries/lorawan-stack-docs/ and https://github.com/TheThingsIndustries/lorawan-stack-aws/ so that we can add this to the documentation and deployments.

File: pkg/webui/lib/shared-messages.js [hunk 1]
```diff
@@ -366,6 +366,7 @@ export default defineMessages({
   status: 'Status',
   statusDescription: 'The status of this gateway may be visible to other users',
   statusUnknown: 'Status unknown',
+  statusPage: 'Status Page',
```
Comment (kschiffer) [inline pkg/webui/lib/shared-messages.js]:
```suggestion
  statusPage: 'Status page',
```
No title case.

File: pkg/webui/console/views/admin-packet-broker/messages.js [hunk 2]
```diff
@@ -68,4 +68,5 @@ export default defineMessages({
   gatewayVisibilityInformation:
     'You can use the checkboxes to control what information of your gateways will be visible. Note that this information will be visible to the public and not only to registered networks.',
   defaultGatewayVisibilitySet: 'Default gateway visibility set',
+  packetBrokerStatusPage: 'Packet Broker Status Page',
```
Comment (kschiffer) [inline pkg/webui/console/views/admin-packet-broker/messages.js]:
```suggestion
  packetBrokerStatusPage: 'Packet Broker status page',
```
No title case.

**Review by kschiffer — APPROVED** [review]
LGTM. See my remarks.

**Pull request merged**